.github/workflows/ci.yml

```diff
@@ -6,8 +6,8 @@ on:
       - master
   pull_request:
   schedule:
-    # Run at 18:00 UTC every day
-    - cron: '0 18 * * *'
+    # Run multiple times a day as the successfull cached links are not checked every time.
+    - cron: '0 */4 * * *'
```

Comment:
Definitely willing to try running more than once per day, but I feel like 2 or 3 times per day should be enough. This will run six times a day, which seems excessive to me.

Reply:
Sure, let's run it then 3x a day.